### ISSUE TITLE
Put downloadable files in folders with version number in SF.net

### ISSUE THREAD

**Issue by Fellypao** [highperformancecoder/scidavis-old#60]
Please resume the old file hosting scheme on SourceFoge.net, where each version of SciDAVis was saved in a folder with the version number, because this will make it easier for people searching for SciDAVis directly there.

**Comment by highperformancecoder** [comment]
On Thu, Oct 26, 2017 at 12:10:05PM +0000, Fellype do Nascimento wrote:
> Please resume the old file hosting scheme on SourceFoge.net, where each version of SciDAVis was saved in a folder with the version number, because this will make it easier for people searching for SciDAVis directly there.

OK - doing that now. I stopped doing that before because on some other
projects I manage, I ended up with a lot of directories with only one
file in it. And also, I needed to remember to add the README.md file
about linux distros. But scidavis has 3 files per release, so perhaps
it makes more sense. But I still need to remember to copy README.md
:).

Cheers


-- 

----------------------------------------------------------------------------
Dr Russell Standish                    Phone 0425 253119 (mobile)
Principal, High Performance Coders
Visiting Senior Research Fellow        hpcoder@hpcoders.com.au
Economics, Kingston University         http://www.hpcoders.com.au
----------------------------------------------------------------------------
